sandpiper/processes/wps_evaluate_rule.py

```diff
@@ -81,10 +82,17 @@ def _handler(self, request, response):
             process_step="start",
         )
 
-        with open(parse_tree_path) as json_file:
-            parse_tree = json.load(json_file)
-        with open(variables_path) as json_file:
-            collected_variables = json.load(json_file)
+        try:
+            file_ = "parse tree"
```

Comment:
Not sure I'm a huge fan of defining this variable _just_ to use in the error message if it comes up. Could we not just use the `_path` variables instead?

Reply:
Ya I wasn't sure about this, but wanted to see what you would say. Only thing is that the actual variable names have '_' in them which won't show up in the ProcessError. I can try some other things out.